<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="INHYNVPADIGELA1" name="testPack.Basic" tests="1" failures="0" timestamp="20 Apr 2018 02:40:51 GMT" time="0.001" errors="1">
  <testcase name="f" time="0.001" classname="testPack.Basic">
    <error type="java.lang.Error" message="Unresolved compilation problems: 
	Syntax error, insert &quot;VariableDeclarators&quot; to complete LocalVariableDeclaration
	Syntax error, insert &quot;;&quot; to complete LocalVariableDeclarationStatement
	dfgdfgdfg cannot be resolved
">
      <![CDATA[java.lang.Error: Unresolved compilation problems: 
	Syntax error, insert "VariableDeclarators" to complete LocalVariableDeclaration
	Syntax error, insert ";" to complete LocalVariableDeclarationStatement
	dfgdfgdfg cannot be resolved

	at testPack.Basic.f(Basic.java:8)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:644)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:850)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1176)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:128)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1299)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1224)
	at org.testng.TestNG.runSuites(TestNG.java:1139)
	at org.testng.TestNG.run(TestNG.java:1110)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </error>
  </testcase> <!-- f -->
</testsuite> <!-- testPack.Basic -->
